deep_water_level/src/infer.py

```diff
@@ -132,7 +132,9 @@ def t2s(t):
         loss += mse
         n_images += len(images)
 
-    print(f"Dataset: {dataset_dir}, Average loss: {loss / len(dataset)}, images: {n_images}, dataset size: {len(dataset)}")
+    print(
```

Comment:
Not sure why some format things are changing 🤷

Reply:
Hopefully it is reaching an equilibrium :-)